conda/recipes/rmm/conda_build_config.yaml

```diff
@@ -1,17 +1,18 @@
 c_compiler_version:
-  - 11
+  - 13  # [not os.environ.get("RAPIDS_CUDA_VERSION", "").startswith("11")]
+  - 11  # [os.environ.get("RAPIDS_CUDA_VERSION", "").startswith("11")]
 
 cxx_compiler_version:
-  - 11
+  - 13  # [not os.environ.get("RAPIDS_CUDA_VERSION", "").startswith("11")]
+  - 11  # [os.environ.get("RAPIDS_CUDA_VERSION", "").startswith("11")]
 
 cuda_compiler:
-  - cuda-nvcc
-
-cuda11_compiler:
-  - nvcc
+  - cuda-nvcc  # [not os.environ.get("RAPIDS_CUDA_VERSION", "").startswith("11")]
+  - nvcc  # [os.environ.get("RAPIDS_CUDA_VERSION", "").startswith("11")]
```

Comment:
This change is optional -- we could leave it as `cuda11_compiler` / `cuda_compiler` and not make the corresponding change in `meta.yaml` (leave `{{ compiler('cuda11') }}` in place).

Reply:
I like this change, keeping more of the future compiler-version churn concentrated in the git blame in `conda_build_config.yaml`.